addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,540 @@
+	displayName = "Rifleman";
+	primaryWeapon[] = {
+		"CUP_arifle_AKS74",
+		"CUP_arifle_AKS74_Early"
+	};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_545x39_AK_M"),
+		LIST_2("CUP_30Rnd_TE1_Green_Tracer_545x39_AK74_plum_M"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	secondaryWeapon[] = {"CUP_launch_RPG7V"};
+	secondaryAttachments[] = {"cup_optic_pgo7v2"};
+	backPack[] = {"CUP_B_CivPack_WDL"};
+	magazines[] += {LIST_3("CUP_OG7_M")};
+};
+
+class arg : r {
+	displayName = "Assistant RPG7V2 Grenadier";
+	backPack[] = {"CUP_B_CivPack_WDL"};
+	backpackItems[] = {
+		LIST_3("CUP_OG7_M")
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	secondaryWeapon[] = {};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : car {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_arifle_AKS74U"};
+	magazines[] = {
+		LIST_5("CUP_30Rnd_545x39_AK_M"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		"SmokeShell"
+	};
+};
+
+class ftl : r {
+	displayName = "Fireteam Leader";
+	primaryWeapon[] = {
+		"CUP_arifle_AKS74_GL",
+		"CUP_arifle_AKS74_GL_Early"
+	};
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	magazines[] += {
+		LIST_15("CUP_1Rnd_HE_GP25_M"),
+		LIST_2("CUP_1Rnd_SMOKE_GP25_M")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"Binocular"};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	primaryWeapon[] = {
+		"CUP_arifle_AKS74_GL",
+		"CUP_arifle_AKS74_GL_Early"
+	};
+	sidearmWeapon[] = {"CUP_hgun_PMM"};
+	magazines[] += {
+		LIST_2("CUP_IlumFlareRed_GP25_M"),
+		LIST_2("CUP_1Rnd_SmokeRed_GP25_M"),
+		LIST_3("CUP_12Rnd_9x18_PMM_M")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_RUS_Beret_VDV"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_st138_prc77"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"CUP_arifle_RPK74_45"};
+	secondaryWeapon[] = {};
+	backPack[] = {"CUP_B_CivPack_WDL"};
+	magazines[] = {
+		LIST_16("CUP_45Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class rat : car {
+	displayName = "Rifleman (Anti-Tank)";
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"CUP_srifle_SVD"};
+	scope[] = {"cup_optic_pso_1"};
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
```

Comment:
RPG-18 on the marksman?

Reply:
Yes, apparently according to the reference document (verified source) the designated marksman is equipped with both an SVD as well as an RPG 18/22. I guess they have lots of organic firepower?